e2e_playwright/st_alert_test.py

```diff
@@ -56,3 +56,13 @@ def test_alerts_rendering(themed_app: Page, assert_snapshot: ImageCompareFunctio
 def test_check_top_level_class(app: Page):
     """Check that the top level class is correctly set."""
     check_top_level_class(app, "stAlert")
+
+
+def test_material_symbol_from_latest_font_version_rendering(
+    themed_app: Page, assert_snapshot: ImageCompareFunction
```

Comment:
nit: I think it would be fine to just use `app` here to test whether the new icons work without running it via dark mode as well.